### PR TITLE
refactor: Rename "Content" to "Activity"

### PR DESCRIPTION
**Pull request by magicus**
Have a look. What do you say? I think this is a definite improvement.

### CODE REVIEW

File: common/src/main/java/com/wynntils/models/activities/ContentBookQueries.java [hunk 1]
```diff
@@ -272,15 +272,15 @@ protected void toggleTracking(String name, ContentType contentType) {
         query.executeQuery();
     }
 
-    private int findTrackedContent(ContainerContent container, String name, ContentType contentType) {
+    private int findTrackedContent(ContainerContent container, String name, ActivityType activityType) {
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/models/activities/ContentBookQueries.java]: Content is actually rooted a bit deeper than just types.

File: common/src/main/java/com/wynntils/models/activities/ActivityModel.java [hunk 2]
```diff
@@ -40,7 +40,7 @@
 import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-public final class ContentModel extends Model {
+public final class ActivityModel extends Model {
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/models/activities/ActivityModel.java:50]: Add a comment saying this is wynn's content book basically?

**Review by kristofbolyai — COMMENTED** [review]
I only fear that this would be harder to find for new developers, who don't know this naming.

File: common/src/main/java/com/wynntils/models/activities/ContentBookQueries.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/models/activities/ContentBookQueries.java]: This is sooo weird! I searched for "content" in IntelliJ, and it does not show this! I have checked again and again now, it shows this if you open in the Find window, but not in the normal list. I have not selected any case or complete word filters. It just ignores the entire ContentBookQueries.java file.

File: common/src/main/java/com/wynntils/models/activities/ActivityModel.java (hunk 2 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/models/activities/ActivityModel.java:50]: Good point.

**Comment by magicus** [comment]
> I only fear that this would be harder to find for new developers, who don't know this naming.

I agree that this is a potential problem. But then again, we have other very specific terms that people will need to learn, like "model" and "manager", so it is one of many.

**Comment by magicus** [comment]
Note that I have not renamed features at the time. I am not sure if we should do that, but if so, we need to make config upfixers.

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**